### ISSUE TITLE
Arrays defined using type expression shortcut are not displayed ok

### ISSUE THREAD

**Issue by nmarinel01** [mulesoft/api-console#342]
Develop branch
Having created the following RAML file :

```
#%RAML 1.0
title: API with types
types:
  Email: # normal object type declaration
    type: object
    properties:
      subject: string
      body: string
  EmailsLong: # array type declaration
    type: array
    items: Email
    minItems: 1
    uniqueItems: true
  EmailsShort: # array type declaration using type expression shortcut
    type: Email[] 
    minItems: 1
    uniqueItems: true
    example: # example that contains array
      - # start item 1
        subject: My Email 1
        body: This is the text for email 1.
      - # start item 2
        subject: My Email 2
        body: This is the text for email 2.
/mail:
  get:
    responses:
      200:
        body:
          application/json:
            type: EmailsShort
```

At console, EmailsShort type is not displayed correcty and neither its usage at resource response body
![image](https://cloud.githubusercontent.com/assets/19980188/19312431/b1f5036a-9068-11e6-87f9-9d95e00d9e5b.png)

![image](https://cloud.githubusercontent.com/assets/19980188/19312448/be745adc-9068-11e6-84ba-92d6f31d5f7c.png)

**Comment by sichvoge** [comment]
In version 3.0.15 it gets displayed, but the last problem still remains

![image](https://cloud.githubusercontent.com/assets/498114/24257762/966df02a-0fe3-11e7-87f2-06ec27ecb4f1.png)

What should be the expected behaviour here? An entry for what the type of the items are?

**Comment by carowright** [comment]
Fix available in [v3.0.17](https://github.com/mulesoft/api-console/releases/tag/v3.0.17)
